apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -277,32 +273,45 @@ const CustomizableCurriculumCatalogCard = ({
               )}
             >
               <Button
-                color={quickViewButtonColor}
+                color={Button.ButtonColor.neutralDark}
                 type="button"
                 onClick={onQuickViewClick}
                 aria-label={quickViewButtonDescription}
                 text={i18n.quickView()}
                 className={`${style.buttonFlex} ${style.quickViewButton}`}
               />
-              <Button
-                __useDeprecatedTag
-                color={quickViewButtonColor}
-                type="button"
-                href={pathToCourse}
-                aria-label={i18n.learnMoreDescription({
-                  course_name: courseDisplayName,
-                })}
-                text={i18n.learnMore()}
-                className={`${style.buttonFlex} ${style.learnMoreButton}`}
-              />
-              {(isSignedOut || isTeacher) && (
+              {isTeacherOrSignedOut && (
+                <>
+                  <Button
+                    __useDeprecatedTag
+                    color={Button.ButtonColor.neutralDark}
+                    type="button"
+                    href={pathToCourse}
+                    aria-label={i18n.learnMoreDescription({
+                      course_name: courseDisplayName,
+                    })}
+                    text={i18n.learnMore()}
+                    className={`${style.buttonFlex} ${style.teacherAndSignedOutLearnMoreButton}`}
+                  />
+                  <Button
+                    color={Button.ButtonColor.brandSecondaryDefault}
+                    type="button"
+                    onClick={() => handleClickAssign('top-card')}
+                    aria-label={assignButtonDescription}
+                    text={assignButtonText}
+                    className={style.buttonFlex}
+                  />
+                </>
+              )}
+              {!isTeacherOrSignedOut && (
                 <Button
+                  __useDeprecatedTag
                   color={Button.ButtonColor.brandSecondaryDefault}
                   type="button"
-                  onClick={() => handleClickAssign('top-card')}
-                  aria-label={assignButtonDescription}
-                  text={assignButtonText}
-                  className={style.buttonFlex}
+                  href={pathToCourse}
+                  aria-label={i18n.tryNow()}
```

Comment:
Non-blocking: can this aria label show the course name with `course_name: courseDisplayName,` too?